api/src/test/java/org/motechproject/nms/api/osgi/KilkariControllerBundleIT.java

```diff
@@ -272,6 +279,25 @@ public void testCreateSubscriptionRequestDifferentPacks() throws IOException, In
     }
 
     @Test
+    public void testCreateSubscriptionsNoLanguageInDB() throws IOException, InterruptedException {
+        setupData();
```

Comment:
We could be making the same point here but every test (except for 3, which would be affected anyways) call setupData here, which in turn drops the data and creates new stuff. I'm just saying that it is good practice to refactor common stuff into a 'before' if the majority of the tests need to do the same thing.